config/m4/ucx.m4

```diff
@@ -67,21 +67,31 @@ AS_IF([test "x$ucx_checked" != "xyes"],[
         ],
         [])
 
+        AS_IF([test "x$ucx_happy" = "xyes"],
```

Comment:
If statement duplicated in line 88, perhaps combine?

Reply:
it's not duplicate, during version check ucx_happy might become "no", then we don't need to do following checks
```
            AS_IF([test $ucx_major -lt ${UCX_MIN_REQUIRED_MAJOR} && test $ucx_minor -lt ${UCX_MIN_REQUIRED_MINOR}],
            [
                AC_MSG_ERROR([Required UCX version: ${UCX_MIN_REQUIRED_MAJOR}.${UCX_MIN_REQUIRED_MINOR}])
                ucx_happy="no"
            ], [])
```